project.toml

```diff
@@ -0,0 +1,4 @@
+[[build.env]]
+name="NEWRELIC_PYTHON_VERSION"
+value="8.1.0.180"
+
```

Comment:
Project.toml is for users to put into their applications.

Remove this file, but add a section [like this](https://github.com/paketo-buildpacks/new-relic/blob/main/buildpack.toml#L34-L37) to buildpack.toml. That is where the ConfigurationResolver will look and where you can set the default value. That said, you probably don't want to put an exact default value like this or it's going to get out of date very quick. Usually we like to have something like default version being a major version or possibly a major.minor if it doesn't move too quickly.

Reply:
I have decided to allow the users to install the New Relic agent using paketo/python-pip buildpack by configuring their requirements.txt in route of their app. That said it makes this configuration redundant.

Reply:
OK, that makes perfect sense to me. I would say just delete this file.